### DISCUSSION TITLE
Where are you hold your Postgre DB with transactions?

### DISCUSSION THREAD

**Discussion by neuroborus** [blockscout/blockscout#7314] [Q&A]
Provide me please with location of PostgreSQL database

As I understand It places loacally in container docker-compose-db.yml?
And if I want to save my database after destroying of container, I need to write "volumes" in this compose file?

**Comment by vbaranov** [comment]
If you're talking about docker compose setup, db files are created in Docker-related path.

I added explicit volume into `./docker-compose/services/blockscout-db-data` in 
https://github.com/blockscout/blockscout/pull/7332/files#diff-43215444ec657b1799f38b9d0320e7c5824dc926777873dbe77f7b7e00de4e7dR15-R16. You could pull update from master branch and DB will be generated at that path.
